dashboards/Gamestudio-qs/Alerts/CPU-usage.yml

```diff
@@ -0,0 +1,40 @@
+---
+# Name of the alert
+name: CPU usage alert
```

Comment:
Could you make this name a little more descriptive? These can be split out from the quickstart, so it's good to have a name that can stand on its own.